integr8ly/tutorial-web-app
# INTLY-871: Fix About modal

## Motivation
Fixes INTLY-871 - issue with intermittent ability to open and close About modal.

## What
Fixes the app's ability to recognize the About modal open/close state.

## Why
About box was opening and closing but not consistently (only on every other open).

## Verification Steps
1. Go to user menu and click About.
2. Click the upper-right hand 'x' to close the About box.
3. Repeat a second time and verify that the About box displays/closes.

## Checklist:

## Progress
